CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 [//]: # (current developments)
 
+## 23.3.1 (2023-03-28)
+
+### Enhancements
+
+* Fix and re-enable binstar tests. Replace custom property caching with `functools.cached_property`. (#12495)
+
+### Bug fixes
+
+* Restore default argument for `SubdirData` method used by `conda-index`. (#12513)
+* Include `conda.gateways.repodata.jlap` submodule in package. (#12545)
+
+### Other
+
+* Add linux-s390x to multi-arch ci/dev container. (#12498)
+* Expose a `MINIO_RELEASE` environment variable to provide a way to pin `minio` versions in CI setup scripts. (#12525)
+* Add `jsonpatch` dependency to support `--experimental=jlap` feature. (#12528)
+* Release with a version of rever that does not double-count commits for
+  aliased committers.
```

Comment:
While these commit counts are a nice to have, they're not important enough to make releasing more complicated by requiring us to checkout from rever's master to get them. I'm ok with leaving the counts as is in `.authors.yml` but announcing it in the changelog is unnecessary.

```suggestion
```